### PR TITLE
Fixed ionic tutorial chapters: 4,5,6

### PR DESCRIPTION
**Pull request by Vercryger**
I found some errors with the snippets, so i fixed to avoid future confusions.

### CODE REVIEW

File: .docs/angular-meteor/packages/tutorial-step-diff-compiler/.npm/plugin/mdg:tutorial-step-diffs-plugin/npm-shrinkwrap.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "dependencies": {
     "am-git-patch-parser": {
-      "version": "0.2.1"
+      "version": "http://registry.npmjs.org/am-git-patch-parser/-/am-git-patch-parser-0.2.1.tgz"
```
Comment (Urigo) [inline .docs/angular-meteor/packages/tutorial-step-diff-compiler/.npm/plugin/mdg:tutorial-step-diffs-plugin/npm-shrinkwrap.json]: @Vercryger why this?

Comment (Vercryger) [inline .docs/angular-meteor/packages/tutorial-step-diff-compiler/.npm/plugin/mdg:tutorial-step-diffs-plugin/npm-shrinkwrap.json]: I don't know why exactly, my linux os change that automatically i guess.
I change that, i can provide a new pr if you want. Let me know.

Comment (Urigo) [inline .docs/angular-meteor/packages/tutorial-step-diff-compiler/.npm/plugin/mdg:tutorial-step-diffs-plugin/npm-shrinkwrap.json]: Yes let's do one without that

**Comment by Urigo** [comment]
Also, please rebase against the latest master, some of it got fixed already (`add` instead of `remove`)

**Comment by Vercryger** [comment]
Done!

**Pull request merged**